src/terra-dev-site/ss/filler17/PlaceholderPage.module.scss

```diff
@@ -0,0 +1,19 @@
+:local {
```

Comment:
Could these styles sheets be shared?

Reply:
Yeah pasting the same style sheet over and over was not the most efficient strategy for sure. Fixed this when addressing Matt's comment about using an mdx file instead: https://github.com/cerner/terra-dev-site/pull/217/commits/c52489d70cfbf8073bc1839a448f19f3bef40f4d